sale_commercial_partner/views/sale.xml

```diff
@@ -0,0 +1,27 @@
+<odoo>
```

Comment:
Please rename file to sale_views.xml https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming